kontrakter/siktelseTiltale/1.0/siktelseTiltale.schema.json

```diff
@@ -0,0 +1,870 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
@calle-777:
Trenger vi egentlig data om personer der data er likt med folkeregister.
Hadde det vært bedre å bare sende fødselsnummer i disse tilfellene?
Om det er diff er det dataen fra fra begjæring som gjelder
Om data endres i domstolen

Reply:
Om data endres i domstolen så blir det presentert for brukeren og det må en manuell retting til.
Foreløpig blir person som den er med data selv om det er likt som folkeregisteret, vi får se på ekstra sjekk hos oss og endring senere.